packages/fxa-auth-server/lib/senders/emails/renderer.ts

```diff
@@ -8,6 +8,8 @@ import * as templates from './templates';
 import * as layouts from './layouts';
 import path = require('path');
 const config = require('../../../config').getProperties();
+import { compileSassToCss } from './sass-compile-files';
```

Comment:
When the tests for scripts are being run, the `emails-scss` step doesn't get executed as part of the `write-emails-to-disk` test and hence the validation errors were being thrown. Executing the compilation step will ensure sass files are compiled down to css irrespective of from where its being called.

Reply:
Ah interesting, nice find. 🙂 I'm glad we know why CI was choking on this now.

I'm OK with leaving this here for now since this makes CI happy and this PR is about RTL styles if this isn't quick to change, but I think we can compile all of the SCSS to CSS at the auth-server build step. We don't need SCSS files except during development.

Reply:
Ahhh okay so it's okay if the scss are not compiled down to css during testing